Makefile

```diff
@@ -58,20 +58,25 @@ bin/aws-fsx-csi-driver: | bin
 .PHONY: all
 all: all-image-docker
 
+# Builds all images and pushes them
 .PHONY: all-push
-all-push:
-	docker buildx build \
-		--no-cache-filter=linux-amazon \
-		--platform=$(PLATFORM) \
-		--progress=plain \
-		--target=$(OS)-$(OSVERSION) \
-		--output=type=registry \
-		-t=$(IMAGE):$(TAG) \
-		.
-	touch $@
+all-push: all-image-registry push-manifest
```

Comment:
I think the `push-manifest` path leads to do everything `all-image-registry` in addition to also making the manifest. Is that expected? I don't think there's any harm in it but it might be worth testing having just `all-push: push-manifest `

Reply:
Alternatively - having `push-manifest` make the image and the manifest is a bit stretched for the name. You can change the path for `push-manifest` to not call `all-image-registry`. If not, could we change the name to something more encompassing like `all-image-manifest`? I know it's not a great name haha please feel free to come up with a better one